test/integration/controller/jobset_controller_test.go

```diff
@@ -1499,6 +1575,26 @@ func checkNoActiveJobs(js *jobset.JobSet, numFinishedJobs int) {
 	}, timeout, interval).Should(gomega.Equal(true))
 }
 
+// Check that the jobs' deletion timestamp is set or not set for the provided number of jobs.
+func checkJobsDeletionTimestamp(js *jobset.JobSet, set bool, numJobs int) {
```

Comment:
can we please move this to https://github.com/kubernetes-sigs/jobset/blob/main/test/util/util.go

Reply:
I also suggest to use the format: `expectJobsDeletionTimestampSet` and `expectJobsDeletionTimestampNotSet` instead of passing a flag.